### PR TITLE
✉️ 🛩 mailjet implementation

### PR DESCRIPTION
**Pull request by DerCalli**
**Description:**

- Skeleton for Mailjet-Mail-Provider-Plugin/Mailjet-Mailer, anyone fine with the structure and naming? I would like to have your opinion first before I have to adjust everything again afterwards. 😉

### CODE REVIEW

File: bundles/mailjet-mail-connector/.github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+jobs:
+  continous-integration:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: [ "7.3", "7.4" ]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```
Comment (sourcecube) [inline bundles/mailjet-mail-connector/.github/workflows/main.yml]: `checkout@v3`

File: bundles/mailjet-mail-connector/.github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        uses: shivammathur/setup-php@v2 #https://github.com/shivammathur/setup-php
+        with:
+          php-version: ${{ matrix.php-versions }}
+          extensions: mbstring, dom, fileinfo, pgsql, json
+          coverage: xdebug #optional
+      - name: Get composer cache directory
+        id: composercache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+      - name: Cache composer dependencies
+        uses: actions/cache@v2
```
Comment (sourcecube) [inline bundles/mailjet-mail-connector/.github/workflows/main.yml]: `cache@v3`

File: bundles/mailjet-mail-connector/.github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  push:
+    branches: [ main ]
+
+jobs:
+  continous-integration:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php-versions: [ "7.3", "7.4" ]
```
Comment (sourcecube) [inline bundles/mailjet-mail-connector/.github/workflows/main.yml]: Should this be 8.0 if the package only supports 8.0

File: bundles/mailjet-mail-connector/src/FondOfOryx/Zed/MailjetMailConnector/MailjetMailConnectorDependencyProvider.php [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        return $container;
+    }
+
+    /**
+     * @return \Spryker\Zed\MailExtension\Dependency\Plugin\MailProviderPluginInterface
+     */
+    protected function getMailer(): MailProviderPluginInterface
+    {
+        return new MailjetMailer(
+            new MailjetClient(
```
Comment (sourcecube) [inline bundles/mailjet-mail-connector/src/FondOfOryx/Zed/MailjetMailConnector/MailjetMailConnectorDependencyProvider.php]: If the client supports additional config I would like to see some configuration for that as well

**Review by sourcecube — APPROVED** [review]

File: bundles/mailjet-mail-connector/README.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# No payment Credit Memo Module
```
Comment (SpiderDan98) [inline bundles/mailjet-mail-connector/README.md]: Correct title?

**Review by SpiderDan98 — APPROVED** [review]

File: bundles/mailjet-mail-connector/.github/workflows/main.yml (hunk 3 above)
Comment (julianhyatt) [inline bundles/mailjet-mail-connector/.github/workflows/main.yml]: you are using php >= 8.0 and use 7.3 and 7.4 in matrix?

**Review by julianhyatt — APPROVED** [review]

**Pull request merged**